packages/os/os.spec

```diff
@@ -698,7 +690,7 @@ if [ -s "%{_cross_repo_root_json}" ] ; then
 fi
 
 install -d %{buildroot}%{_cross_templatedir}
-install -p -m 0644 %{S:5} %{S:6} %{S:7} %{S:8} %{S:14} %{S:15} %{S:16} %{S:17} %{S:18} %{S:19} %{S:21} \
+install -p -m 0644 %{S:5} %{S:6} %{S:7} %{S:14} %{S:15} %{S:16} %{S:17} %{S:18} %{S:19} %{S:21} \
```

Comment:
If `S:8` isn't installed anymore, you have to remove the file from the sources as well. That file is `oci-default-hooks-json`